accounts/abi/bind/base.go

```diff
@@ -289,12 +289,7 @@ func (c *BoundContract) transact(opts *TransactOpts, contract *common.Address, i
 	if opts.Signer == nil {
 		return nil, errors.New("no signer to authorize the transaction with")
 	}
-	var signedTx *types.Transaction
-	if rawTx.IsPrivate() {
```

Comment:
where is this check done now?

Reply:
should we remove this commented code?

Reply:
it's done here at many places: https://github.com/ConsenSys/quorum/pull/1223/files#diff-18aeac66ddb948fefafc8c96289984cd90c94c54d0616e96165ffff8865b6250R73

the signer is not passed as argument anymore: https://github.com/ethereum/go-ethereum/pull/21356/files#diff-85ec70c4e3549eb9e475493078d46953b17ae6c3263c31fbe94327fbcdc37ad1R259